tests/kernel_performance/install_qatestset.pm

```diff
@@ -69,6 +69,12 @@ sub setup_environment {
         assert_script_run("/usr/share/qa/qaset/bin/deploy_hana_perf.sh HANA $mitigation_switch $qaset_kernel_tag");
         assert_script_run("ls /root/qaset/deploy_hana_perf_env.done");
 
+        # workaround to prevent network interface random order
+        my $project_m_role = get_var("PROJECT_M_ROLE", "");
```

Comment:
Is it `if(check_var("PROJECT_M_ROLE", "PROJECT_M_ABAP"))` a suitable alternative?

Reply:
+1 to this one. I'd say you can replace lines 73 to 76 with:
```
assert_script_run('dracut -f --add-drivers "tg3"') if (check_var('PROJECT_M_ROLE', 'PROJECT_M_ABAP'));
```

Reply:
fixed, thank you all!